conformance/findJava.sh

```diff
@@ -0,0 +1 @@
+../test/findJava.sh
```

Comment:
Why do we need this?

Reply:
fixed. (changed the relative path in `test/balt-sub.mk` to work in both `test` and `conformance` directories)